docs/fedora-releases-guide/index.md

```diff
@@ -253,3 +253,28 @@ you can override this behaviour by specifying
 `allowed_builders` in the [job configuration](/docs/configuration/downstream/bodhi_update).
 
 For a configuration example and retriggering, see [dist-git onboarding](./dist-git-onboarding.md#bodhi-update-job).
+
+
+# Permission Handling
+
+The downstream automation is a chain of jobs: propose downstream/pull from upstream -> koji build -> bodhi update.
+
+This chain can break (resulting in no followup job) if the permissions are not configured correctly in the Packit configuration.
+
+### Propose downstream/Pull from upstream -> Koji build
+
+A Koji build can be triggered by a merged pull request, a direct push or a comment.
+
+* If you only merge Packit created pull requests, you don't need to specify other settings.
+* If you create your own pull request and you want Packit to automatically trigger a Koji build when you merge it you need to be in the [`allowed_pr_authors`](https://packit.dev/docs/configuration/downstream/koji_build#optional-parameters) list, even when you are reusing Packit commits.
```

Comment:
```suggestion
* If you open your own pull request and you want Packit to automatically trigger a Koji build when you merge it you need to be in the [`allowed_pr_authors`](https://packit.dev/docs/configuration/downstream/koji_build#optional-parameters) list, even in case the PR contains commits created by Packit.
```